dev/tasks/utils/tools.js

```diff
@@ -136,5 +138,120 @@ module.exports = {
 		}
 
 		return ret.output;
+	},
+
+	/**
+	 * Links directory located in source path to directory located in destination path using `ln -s` command.
+	 * @param {String} source
+	 * @param {String} destination
+	 */
+	linkDirectories( source, destination ) {
+		// Remove destination directory if exists.
+		if ( this.isDirectory( destination ) ) {
+			this.shExec( `rm -rf ${ destination }` );
+		}
+
+		this.shExec( `ln -s ${ source } ${ destination }` );
```

Comment:
Please check with @mlewand that it's going to work on Windows. If not, perhaps we should use some npm module that would gracefully degrade to Windows env.